addons/account/models/partner.py

```diff
@@ -506,8 +515,27 @@ def _get_company_currency(self):
     invoice_warn_msg = fields.Text('Message for Invoice')
     # Computed fields to order the partners as suppliers/customers according to the
     # amount of their generated incoming/outgoing account moves
+<<<<<<< HEAD
```

Comment:
hmmm :)

Reply:
oup. It's ok, it's draft. 

Didn't took a deep look, just found the old branch it's not fully finalized yet